modules/common/services/audio.nix

```diff
@@ -77,6 +132,51 @@ in {
       script = ''${pkgs.pulseaudio}/bin/pa-info > /dev/null 2>&1'';
     };
 
+    # TODO Automate and fix alsa IO ports
+    systemd.services."pipewire-link-starter" = let
```

Comment:
I suppose that we want to aim at dynamic support, i.e., supporting several input/output devices such as bluez_input/output, starting with auto-switch and later selection.